pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -25,6 +32,5 @@ stdenv.mkDerivation rec {
     description = "Build tool that builds code quickly and reliably";
     license = stdenv.lib.licenses.asl20;
     maintainers = [ stdenv.lib.maintainers.philandstuff ];
-    platforms = [ "x86_64-linux" ];
```

Comment:
Couldn't find much relevant info when googling around. So yea, I think it'll have to stay `x86_64-linux`  (this patch didn't change that) :/